components/caseStudyDetails/OtherCaseStudyCard.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { motion } from "framer-motion";
+import { Card } from "@nextui-org/card";
+import Image from "next/image";
+import { Button } from "@nextui-org/button";
+import { Link } from "@nextui-org/link";
+
+import { projectDetails } from "@/constants";
+import { generateFadeInAnimationVariants } from "@/utils";
+
+const OtherCaseStudyCard = ({ project }: { project: string }) => {
+  const otherProjects = Object.keys(projectDetails).filter(
+    (key) => key !== project
+  );
```

Comment:
Great job! This could also be a utility function that you have imported. This would help clean up the code a little.

Reply:
```ts
const OtherCaseStudyCards = ({ project }: { project: string }) => {
  const otherProjects = Object.keys(projectDetails).filter(
    (key) => key !== project <===== the project param
  );
```
I already considered that. But in this code, I need the project param. I'm not sure how to pass the project param when I move that into utils